com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/Messages.java

```diff
@@ -61,6 +61,7 @@
     public static final String APPLICATION_UNBOUND_IN_PARALLEL = "Application {0} was bound to service {1} which was unbound in parallel";
     public static final String ABORT_OPERATION_TIMED_OUT = "Abort operation timed out";
     public static final String FLOWABLE_JOB_RETRY_FAILED = "Flowable job retry failed";
+    public static final String DEPENDENCIES_CANNOT_BE_SATISFIED = "Dependencies for modules {0} cannot be satisfied";

```

Comment:
"Dependencies can't be satisfied" is too generic. If you are going to print an error message make it as descriptive as possible